certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -37,7 +38,7 @@ type operatorData struct {
 	App              string
 	InstallNamespace string
 	TargetNamespace  string
-	InstallModes     map[string]bool
+	InstallModes     map[operatorv1alpha1.InstallModeType]operatorv1alpha1.InstallMode
```

Comment:
Why is the k/v the same type?

Reply:
I think they are different one is a `string` and one is a `struct`, unless I don't understand the question.

[InstallModeType](https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/clusterserviceversion_types.go#L28)

[InstallMode](https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/clusterserviceversion_types.go#L43-L46)

Reply:
@acornett21 is correct. key == InstallModeType, value == InstallMode

Reply:
Whoops. I can't read.